app/furniture/marketplace/product_component.rb

```diff
@@ -13,12 +13,8 @@ def edit_button
       super(title: t("marketplace.products.edit.link_to", name: name), href: location(:edit))
     end
 
-    # 16:9 of 1290 is 1290:725.625
-    # We rounded up.
-    # @see https://www.ios-resolution.com/
-    FULL_WIDTH_16_BY_9 = [1290, 726]
     def hero_image
-      product.photo.variant(resize_to_fill: FULL_WIDTH_16_BY_9)
+      product.photo.variant(resize_to_fill: Media::FULL_WIDTH_16_BY_9)
```

Comment:
Would it make sense for this code to live in `Media#hero_image` (`Product#photo` is not yet a `Media`, so it doesn't make sense right-here, but it may make sense in the other places)